docs/index.yml

```diff
@@ -38,6 +38,12 @@ landingContent:
             url: project/index.yml
           - text: Word add-ins documentation
             url: word/index.yml
+      - linkListType: learn
+        links: 
+          - text: Introduction to Office client customization with add-ins
+            url: //docs.microsoft.com/learn/modules/intro-office-add-ins/
+          - text: Extend Office clients with Office add-ins
+            url: //docs.microsoft.com/learn/paths/m365-office-add-in-associate/
```

Comment:
```suggestion
            url: /learn/paths/m365-office-add-in-associate/
```

Reply:
Nifty! I didn't realize we could easily link to other doc sets!